.luacov

```diff
@@ -1,33 +1,29 @@
 -- Configuration file for LuaCov
 
 -- This variable is set externally
-local build = os.getenv("BUILD_DIRECTORY") or error("$BUILD_DIRECTORY not set")
-local lib_dir = os.getenv("AWESOME_LIB_DIR") or build .. "lib/"
-local source = os.getenv("SOURCE_DIRECTORY")
+local source = os.getenv("SOURCE_DIRECTORY") or os.getenv('PWD')
+local build = os.getenv("BUILD_DIRECTORY") or (source .. "/build")
+local lib_dir = os.getenv("AWESOME_LIB_DIR") or (source .. "/lib")
```

Comment:
According to the commit message, this should be more than one commit. I have a hard time following all of this. Why the switch to a different version of luacov? Should that perhaps just be unpinned instead? What does `set -v` do and what is its connection with this PR?